### PR TITLE
styled landing page

### PR DESCRIPTION
**Pull request by oagboola**

### CODE REVIEW

File: app/views/home/show.html.haml [hunk 1]
```diff
@@ -1 +1,128 @@
...
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+.col.s12.promotion
```
Comment (MonsieurSlim) [inline app/views/home/show.html.haml:120]: As discussed, footer should be in the layout, either as a partial or in the page itself.

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -1,3 +1,39 @@
 // Place all the styles related to the home controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.categories {
+  background-color: #F0F0F0;
+  text-align: center;
+}
+
+.categories img {
```
Comment (MonsieurSlim) [inline app/assets/stylesheets/home.scss]: Sass allows nexting of styling. Can you look up how this is done?

File: app/views/home/show.html.haml (hunk 1 above)
Comment (oagboola) [inline app/views/home/show.html.haml:120]: okay, I'll change that

File: app/assets/stylesheets/home.scss (hunk 2 above)
Comment (oagboola) [inline app/assets/stylesheets/home.scss]: I'll look it up and make the adjustments

**Pull request merged**